types/JudoTypes.tsx

```diff
@@ -35,8 +35,8 @@ export interface JudoAmount {
 
 export interface JudoReference {
     consumerReference: string,
-    paymentReference: string,
-    metadata?: Map<String, String>,
+    paymentReference?: string,
+    metadata?: Record<string, string>,
```

Comment:
`metadata?: Record<string, any>` ?

Reply:
Still have no idea how to add dynamic values to a Bundle.

Reply:
ok, I'll have a look later this week